spec/github_url_parser_spec.rb

```diff
@@ -60,7 +60,8 @@
       ['scm:git:https://michaelkrog@github.com/michaelkrog/filter4j.git', 'michaelkrog/filter4j'],
       ['github.com/github/combobox-nav', 'github/combobox-nav'],
       ['github.com/hhao785/github.com', 'hhao785/github.com'],
-      ['github.com/contrived_example/githubcom', 'contrived_example/githubcom']
+      ['github.com/contrived_example/githubcom', 'contrived_example/githubcom'],
+      ['scm:git:ssh://github.com/an-organization/a-repository.git/a-repository-subdirectory', 'an-organization/a-repository']
```

Comment:
🎱 Worth having a case for `https://github.com/foobar/yet-another-something-plugin.git` for the case we saw?

Reply:
The string in this test is the same format as the one we saw, but details have been substituted for generic names.

raw string looked like: `scm:git:ssh://github.com/an-organization/a-repository.git/a-repository-subdirectory`
parsing that yielded: `https://github.com/an-organization/a-repository.git`
parsing that parse resulted yielded: `https://github.com/an-organization/a-repository`